src/Utils/Util.cs

```diff
@@ -57,6 +57,36 @@ public static Color TryParseColor(string str, float alpha = 1f)
         return CopyColor(Calc.HexToColor(str.Trim('#')), alpha);
     }
 
+    public static Color HexToColorWithAlphaNonPremultiplied(string hex)
```

Comment:
is it necessary to use nonpremultiplied colours? using premultiplied ones would be more consistent with everest decals, and i don't think having loads of different colour formats is a good idea